core/src/main/scala/io/finch/response/package.scala

```diff
@@ -151,25 +152,28 @@ package object response {
    * An abstraction that is responsible for encoding the response of type `A`.
    */
   trait EncodeResponse[-A] {
-    def apply(rep: A): String
+    def apply(rep: A): Array[Byte]
```

Comment:
What about using `com.twitter.util.Buf` here (for the immutability, if nothing else)?